third_party/libunwindstack/tests/PeCoffEpilogTest.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include "PeCoffEpilog.h"
 
+#include <array>
```

Comment:
Was this change intended? (I quickly skimmed through the file and didn't see any usage of std::array.)

Reply:
Yes, this is actually used here, none of our builds complain, but the oss-fuzz build doesn't work on my machine because of this missing include.

Reply:
Cool. Thanks for clarifying!